internal/graph/teams.resolvers.go

```diff
@@ -1296,6 +1296,75 @@ func (r *teamResolver) VulnerabilitiesSummary(ctx context.Context, obj *model.Te
 	return retVal, nil
 }
 
+// VulnerabilityMetrics is the resolver for the vulnerabilityMetrics field.
+func (r *teamResolver) VulnerabilityMetrics(ctx context.Context, obj *model.Team, from *scalar.Date, to *scalar.Date, environment *string) (*model.VulnerabilityMetrics, error) {
+	var metrics []*model.VulnerabilityMetric
+
+	err := ValidateDateInterval(*from, *to)
```

Comment:
Dersom `to` og `from` er optional, så må vi håndtere at de er `nil`. Nå vil denne kaste en nil-panic dersom de ikke er satt

Reply:
Endrer til not null i skjema